drivers/charger/sbs_charger/sbs_charger.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2023 Cirrus Logic, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_SBS_CHARGER_H_
+#define ZEPHYR_DRIVERS_SENSOR_SBS_CHARGER_H_
+
+#include <stdint.h>
+#include <zephyr/drivers/i2c.h>
+
+#define SBS_CHARGER_REG_SPEC_INFO     0x11
+#define SBS_CHARGER_REG_STATUS	      0x13
+#define SBS_CHARGER_REG_ALARM_WARNING 0x16
+
+#define SBS_CHARGER_STATUS_CHARGE_INHIBITED BIT(1)
```

Comment:
BIT(0)?

Reply:
Nice, good catch. I carried this mistake over from the Linux driver. Do you plan on submitting a patch for the Linux driver, if not I can take care of it?

Reply:
I'll submit a patch for the Linux driver. Thanks for asking.